060_Distributed_Search/20_Scan_and_scroll.asciidoc

```diff
@@ -1,14 +1,11 @@
 [[scan-scroll]]
```

Comment:
I wonder if we should update this blockID (and all links pointing to it) because it will show up in the URL, and be weird that it mentions scan?  

But that'd break all existing non-ES inbound links soooo...I dunno.  Thoughts?